### PR TITLE
qf-knm: notes following #2469

### PR DESCRIPTION
**Pull request by fenollp**

### CODE REVIEW

**Comment by fenollp** [comment]
@icehess Could you comment on the leftover note at https://github.com/2600hz/kazoo/pull/2469 ? plz

File: applications/crossbar/src/modules_v2/cb_phone_numbers_v2.erl [hunk 1]
```diff
@@ -29,6 +29,7 @@
 
 -define(CB_LIST, <<"phone_numbers/crossbar_listing">>).
 -define(PORT_NUM_LISTING, <<"port_requests/phone_numbers_listing">>).
+-define(KEY_PORT_NUMBER, 2).
```
Comment (lazedo) [inline applications/crossbar/src/modules_v2/cb_phone_numbers_v2.erl]: @icehess sorry, but can we rename it to `KEY_PORT_NUMBER_IDX_KEY` ? i think its more meaningful.

**Comment by icehess** [comment]
@fenollp how do feel about passing an option like `include_ports` to `knm_number:get` and deal with finding port number in number_manager instead?

**Comment by fenollp** [comment]
@icehess I think I would be ok with that. Some thoughts:

1. Flag would be in extra_options
2. Flag only marks "action", not "state": something returning extra_options() like {outbound, true} should never return this flag. I. E. Use this flag on query, shouldn't appear on reply. 
3. There are other places in knm_number that query port requests. Your code should do the same kind of things

**Pull request merged**